client/app/nasha/add/nasha-unavailable.html

```diff
@@ -0,0 +1,19 @@
+<div class="cui-page">
+    <oui-page-header
+        data-heading="{{::'nasha_order_title' | translate}}">
+    </oui-page-header>
+
+    <div class="cui-page__content nasha-unavailable">
+        <h2 class="oui-header_3" data-translate="nasha_order_unavailable_subtitle"></h3>
+
+        <div class="nasha-unavailable__contact">
+            <div class="nasha-unavailable__image">
+                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><style>.cls-1,.cls-3{fill:none;}.cls-2{clip-path:url(#clip-path);}.cls-3{stroke:#122844;stroke-linecap:round;stroke-linejoin:round;}</style><clipPath id="clip-path" transform="translate(0 0)"><rect class="cls-1" width="32" height="32"/></clipPath></defs><title>phone_32px-bkg-light</title><g id="Calque_2" data-name="Calque 2"><g id="Calque_1-2" data-name="Calque 1"><g id="phone_32px-bkg-light"><g class="cls-2"><g class="cls-2"><path class="cls-3" d="M29.78,22a19.49,19.49,0,0,1-6.2-1,2.12,2.12,0,0,0-1.72.35l-3.79,3.79A26.82,26.82,0,0,1,6.7,13.76L10.49,10a1.37,1.37,0,0,0,.34-1.72,23.15,23.15,0,0,1-.86-6A1.63,1.63,0,0,0,8.25.5h-6A1.63,1.63,0,0,0,.5,2.22,29.25,29.25,0,0,0,29.78,31.5a1.63,1.63,0,0,0,1.72-1.72v-6A1.63,1.63,0,0,0,29.78,22Z" transform="translate(0 0)"/></g></g></g></g></g></svg>
```

Comment:
SVG optimized would be great. (useless `<title>` value, too many groups `<g>` for instance).